lib/std/monotimes.nim

```diff
@@ -15,10 +15,7 @@ meaning that that the following is guaranteed to work:
 ]##
 
 runnableExamples:
-  import std/os
-
   let a = getMonoTime()
-  sleep(10)
```

Comment:
this test made no sense

Reply:
see https://dev.azure.com/nim-lang/Nim/_build/results?buildId=16746&view=logs&jobId=c6054849-8341-5e23-b888-79fd7ec95d3a&j=c6054849-8341-5e23-b888-79fd7ec95d3a&t=8f082e70-3e0a-5206-473a-da5bdc1e1c68

And 
https://github.com/nim-lang/Nim/pull/18149/files#r643817450

Reply:
caused by https://github.com/nim-lang/Nim/issues/18158; I've updated examples accordingly